### ISSUE TITLE
error 'ee key too small' 

### ISSUE THREAD

**Issue by jonsmirl** [UniversalDevicesInc/polyglot-v2#95]
This error is because polyglot is using an old version of TLS which is insecure. Polyglot needs to be updated to TLS 1.2 or better.  Changing SECLEVEL=1 does work, but it works by enabling use of the insecure TLS version on your box which is probably something that you shouldn't do.

**Comment by marcurdy** [comment]
I've made a lot of suggested changes regarding SECLEVEL on ubuntu 20.04, but it's not taking. I agree with jonsmirl emphatically.

**Comment by psmith3** [comment]
I too am seeing this on 2.2.11 running in a Docker container. How can I get around this?

error: Startup error. Shutting down: Error: error:140AB18F:SSL routines:SSL_CTX_use_certificate:ee key too small

**Comment by jimboca** [comment]
Workaround is here https://github.com/UniversalDe...-v2#ssl-error-ee-key-too-small

**Comment by psmith3** [comment]
@jimboca, I am running pulling the binary and creating a docker image. I see the fix you pointed out. I don't think you are using Docker and may not be able to answer my question. Do you know how I could make this change so it is applied in the container? I would rather not make this change after the container is started and instead fix it before the container is created.

**Comment by jimboca** [comment]
Sorry, no idea

**Comment by marcurdy** [comment]
Use Ubuntu 18.04 as the docker base

**Comment by psmith3** [comment]
> Use Ubuntu 18.04 as the docker base
> […](#)

I am running Docker on a Synology NAS. When you say use Ubunti 18.04 as the docker base, are you referring to the FROM in the docker file? I currently am using "FROM debian:buster"

**Comment by marcurdy** [comment]
Yes. Use FROM ubuntu:bionic. It worked well for me as an OS in regards to
providing a compatible version of openssl.

**Comment by psmith3** [comment]
> Yes. Use FROM ubuntu:bionic. It worked well for me as an OS in regards to providing a compatible version of openssl.
> […](#)

Thanks much for this suggestion. Indeed this worked!
